docs/tutorials/scroll.md

```diff
@@ -0,0 +1,130 @@
+---
+section: docs
+layout: docs
+title: Scroll Snap
+---
+
+These examples shows the various scroll snap properties supported by atomic.
+
+### `scroll-behavior` construct
+
+The scroll-behavior CSS property sets the behavior for a scrolling box when scrolling is triggered by the navigation, `CSSOM` scrolling APIs or `SCROLL-SNAP`.
+
+```html
+<div class="Sb(s)">
```

Comment:
I like that the example is simple but the demo below has all the right classes

Reply:
I agree I will make all the example code only have the properties being discussed and have the html with all the required values, so is easer to explain the specific atomic class